### ISSUE TITLE
VS Code 1.53 breaks mermaid preview support

### ISSUE THREAD

**Issue by sebastianzolg** [mjbvz/vscode-markdown-mermaid#98]
It looks like there are some issues with VS code in the latest version. Mermaid diagrams no longer render correctly. 

[1.53 seems to break Markdown Mermaid Support](https://github.com/microsoft/vscode/issues/116805)

VS Code Version: 1.53.2
OS Version: Windows 10
Steps to Reproduce:

Preview markdown document with Mermaid diagram in 1.53.
![image](https://user-images.githubusercontent.com/12109119/109021570-4a278d00-76bb-11eb-858d-3c0aed4a8f48.png)

**Comment by luisee** [comment]
I have the same problem in Windows 10, have you managed to fix the problem? Thanks!

**Comment by sebastianzolg** [comment]
No, @luisee. Still facing the issue. Not sure if somebody monitors this repository.

**Comment by cpshoemake** [comment]
I have the same rendering issue on Windows 10 & VS Code 1.55.2.
